src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,63 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class WorkWithFile {
+    private StringBuilder dataToWrite = new StringBuilder();
```

Comment:
remove this global variable
you should create StringBuilder in the place where you use it

Reply:
ok